lesson22/main.js

```diff
@@ -145,56 +145,85 @@ const createSortBtn = () => {
   return btn;
 };
 
-const renderSortBtn = () => {
-  const btn = createSortBtn();
-  const targetEle = document.getElementById("id");
-  targetEle.appendChild(btn);
+const setDataSort = (usersData) => {
+  Object.keys(usersData[0]).forEach((key) => {
+    if (targetSortElements.includes(key)) {
+      const element = document.getElementById(key);
+      element.dataset.sort = "sort";
+    }
+  });
+};
+
+const renderSortBtn = (usersData) => {
+  setDataSort(usersData);
+  const elements = document.querySelectorAll("[data-sort]");
+  Array.from(elements).forEach((element) => {
```

Comment:
```suggestion
  elements.forEach((element) => {
```

Reply:
I fixed it too.
[4f2be45](https://github.com/yuka830/js-lessons/pull/28/commits/4f2be45f21b29576235b85cfa840a204f85d3288)